### ISSUE TITLE
Move the CSV of Legislators into a daily update

### ISSUE THREAD

**Issue by e01010100** [CoalitionBuilder/okcongress#10]
Currently the CSV file of legislator information has been converted to JSON and committed to our repository. This needs to be moved so that it is pulled from OpenGov on a regular basis and cached. Additionally, there needs to be a way to dump & reset the cache but that doesn't have to be implemented now as someone could DOS the site by constantly hitting that endpoint.

**Comment by bschmoker** [comment]
Ah so basically pull it into the running instance rather than statically including.

Couple things i'd add:
- [ ] does opengov respond with a csv or json?
- [ ] integrity / sanity check of received file
 
> suggest implementing the 'reload functionality' as a grunt task, so it can be run locally but not remotely (nice for debugging, no remote DoS)

**Comment by e01010100** [comment]
yeah, more specifically we want to pull it into a daily cache.
The original source was/is a CSV file saved on github
The list of committees is a Yaml file saved on github except it was a broken yaml file that I had to manually fix. 

In both cases we should do at least some integrity checking but there isn't a whole lot we can do that I can think of off the top of my head.

**Comment by bschmoker** [comment]
yeah i'd imagine a `grunt task` using [this lib](https://www.npmjs.com/package/grunt-periodic) to rebuild the cache every day, and run some checks on random committees / legislators
